docs/java/overview.asciidoc

```diff
@@ -0,0 +1,42 @@
+== Overview
+
+Official low-level client for Elasticsearch. Allows to communicate with an
+Elasticsearch cluster through http. Compatible with any elasticsearch version.
```

Comment:
You have a note about compatibility in the sniffer, maybe also add one here? I suspect is is compatible with **all** versions of Elasticsearch.